src/pkg/cli/compose/loader.go

```diff
@@ -67,6 +67,9 @@ func NewLoader(opts ...LoaderOption) *Loader {
 }
 
 func (c *Loader) LoadProjectName(ctx context.Context) (string, error) {
+	if c == nil {
+		return "", errors.New("missing loader")
```

Comment:
will the user see this? Means nothing to them. Might as well leave panic.

Reply:
Done